pypilot_dependencies/hat/wiringPi.build

```diff
@@ -52,12 +52,12 @@ tar -xf $SRCNAM
 cd $WRKDIR
 
 cd wiringPi
-make DESTDIR=$TMPDIR install
+make DESTDIR=$TMPDIR/usr install
 cd ../devLib
-make DESTDIR=$TMPDIR install
+make DESTDIR=$TMPDIR/usr install
 
 cd ../gpio
-make DESTDIR=$TMPDIR install
```

Comment:
Do you recall why this change was made?   Why add /usr?

Reply:
Because in Makefile PREFIX=/local... So without the /usr it will be installed in $TMPDIR/local/lib...

I could do "make DESTDIR=$TMPDIR PREFIX=/usr/local instead" ...